docs/coordinates/index.rst

```diff
@@ -547,11 +547,6 @@ coordinate systems implemented here include:
 Built-in Frame Classes
 ======================
 
-.. automodapi:: astropy.coordinates.builtin_frames
```

Comment:
I guess a similar question here - certainly, with this removed, the section title should be removed too -- but this particular line is a very big deal in the actual docs!! https://docs.astropy.org/en/latest/coordinates/index.html#built-in-frame-classes